keycloak-export/src/main/resources/theme/ctexport/admin/resources/js/services.js

```diff
@@ -5,14 +5,20 @@ var module = angular.module('keycloak.services', [ 'ngResource', 'ngRoute' ]);
 module.service('Dialog', function($modal) {
     var dialog = {};
 
-    var openDialog = function(title, message, btns, template) {
+    var openDialog = function(title, message, btns, template, sensitive = false, name) {
         var controller = function($scope, $modalInstance, title, message, btns) {
             $scope.title = title;
             $scope.message = message;
             $scope.btns = btns;
+            $scope.sensitive = sensitive;
+            $scope.currentHostname = window.location.hostname;
```

Comment:
$scope.name is already assigned?

Reply:
I'm sorry, what?

Reply:
You are setting values in $scope but not setting $scope.name... and you are using $scope.name a few lines further

Reply:
this has been fixed since